db/migrate/20160209185158_add_stripe_token_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddStripeTokenToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :stripe_token, :string
```

Comment:
I originally named it `stripe_token` as that's how Stripe refers to their response, but I can see how `stripe_customer_id` would make more sense in this context. I can change that.